pylons/chain.json

```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "pylons",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://pylons.tech",
+  "pretty_name": "Pylons",
+  "chain_id": "pylons-mainnet-1",
+  "bech32_prefix": "pylo1",
```

Comment:
Please register a prefix https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
I have submitted a pull request to add the prefix.

Reply:
> Please register a prefix https://github.com/satoshilabs/slips/blob/master/slip-0173.md

https://github.com/satoshilabs/slips/pull/1665

Reply:
Prefix should be changed here to just 'pylo' then.

Reply:
> Prefix should be changed here to just 'pylo' then.

done